model/app/fetcher_http.go

```diff
@@ -118,7 +118,7 @@ func fetchHTTP(src *url.URL, shasum []byte, fs appfs.Copier, man Manifest, prefi
 
 	// For the lock key, we use the checksum when available, and else, we
 	// fallback on the app name.
-	mu := lock.ReadWrite(prefixer.GlobalPrefixer, "app-"+man.Slug()+"-"+man.Checksum())
+	mu := config.GetConfig().Lock.ReadWrite(prefixer.GlobalPrefixer, "app-"+man.Slug()+"-"+man.Checksum())
```

Comment:
Maybe we can add an helper in the `config` package to make those lines shorter and less tedious to read:

```go
config.Lock().ReadWrite(...)
```